ui/apps/platform/src/Containers/Collections/UnsupportedCollectionState.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { ExpandableSection, List, ListItem } from '@patternfly/react-core';
+import { InfoCircleIcon } from '@patternfly/react-icons';
+
+import EmptyStateTemplate from 'Components/PatternFly/EmptyStateTemplate';
+import { CollectionParseError } from './converter';
+
+export type UnsupportedCollectionStateProps = {
+    errors: CollectionParseError['errors'];
+    className?: string;
+};
+
+function UnsupportedCollectionState({ errors, className = '' }: UnsupportedCollectionStateProps) {
+    return (
+        <div className={className}>
+            <EmptyStateTemplate
+                title="This collection cannot be displayed"
+                headingLevel="h2"
+                icon={InfoCircleIcon}
+            >
+                <p className="pf-u-pb-lg">
+                    This collection is valid but cannot be displayed nor edited through the ACS UI
```

Comment:
For product branding, can we replace **the ACS UI** with something like **the user interface**.

Reply:
Although in this case, collapsed by default seems reasonable, I will mention a counter-example where it seems unreasonable and caused me problems yet again yesterday: when an integration test fails because the page throws an exception, `ErrorBoundary` renders the error message as collapsed in the cypress snapshot.